### DISCUSSION TITLE
Set username/password

### DISCUSSION THREAD

**Discussion by Biggtazz** [Lolliedieb/lolMiner-releases#579] [Q&A]
Hello! I'm new to this, so consider this as a laymen's question; 
I set up the miner for nanopool. So far so good, it's mining.
I want to add my e-mailadress somehow, so that Nanopool knows my 'password'. That way I can change some settings. 

I found this in the readme.txt, but I can't get it to work. 

  -u [ --user ] arg                     Wallet or pool user account to mine on
  --pass arg                            Pool user account password (Optional)

Can someone explain to me how to add this rule? Or am I completely thinking the wrong way?

Thanks!

**Comment by jpatel313** [comment]
Add --pass yourPasswordHere in the extra parameters.

ie.
set "EXTRAPARAMETERS=**--pass x123**"
